fe/fe-core/src/test/java/org/apache/doris/catalog/ModifyStorageMediumTest.java

```diff
@@ -0,0 +1,64 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.catalog;
+
+import org.apache.doris.common.DdlException;
+import org.apache.doris.system.Backend;
+import org.apache.doris.system.SystemInfoService;
+import org.apache.doris.thrift.TStorageMedium;
+import org.apache.doris.utframe.TestWithFeService;
+
+import org.junit.jupiter.api.Assertions;
+
+import java.util.List;
+
+public class ModifyStorageMediumTest extends TestWithFeService {
+
+    @Override
+    protected void runBeforeAll() throws Exception {
+        SystemInfoService clusterInfo = Env.getCurrentEnv().getClusterInfo();
+        List<Backend> allBackends = clusterInfo.getAllBackends();
+        // set all backends' storage medium to SSD
+        for (Backend backend : allBackends) {
+            if (backend.hasPathHash()) {
+                backend.getDisks().values().stream()
+                        .peek(diskInfo -> diskInfo.setStorageMedium(TStorageMedium.SSD));
+            }
+        }
+    }
+
+    @Override
+    protected void runAfterAll() throws Exception {
+        Env.getCurrentEnv().clear();
+    }
+
+    public void modifyStorageMediumTest() throws Exception {
```

Comment:
what does the name `modifyStorageMediumTest` come from? you seems not `modify` anything?

Reply:
That's a problem, I will modify it and add some new tests later!